torchvision/transforms/functional.py

```diff
@@ -817,21 +817,25 @@ def rotate(
             Defaults to 0 for all bands. This option is only available for ``pillow>=5.2.0``.
             This option is not supported for Tensor input. Fill value for the area outside the transform in the output
             image is always 0.
+        resample (int, optional): deprecated argument, please use `arg`:interpolation: instead.
 
     Returns:
         PIL Image or Tensor: Rotated image.
 
     .. _filters: https://pillow.readthedocs.io/en/latest/handbook/concepts.html#filters
 
     """
+    if resample is not None:
+        warnings.warn("Argument resample is deprecated. Please, use interpolation instead")
```

Comment:
Should we be honouring the provided `resample` value if `interpolation` is not provided?

Reply:
Do you mean, set `interpolation=resample` if `resample is not None` ? True that it would lead to an unexpected result if called like
```
F.rotate(x, 45, resample=2) == F.rotate(x, 45, interpolation=0)
```

Reply:
Yes exactly. :)